test.js

```diff
@@ -0,0 +1,73 @@
+var tests = {
+
+    	numbers: [1, 5, 66, 224, 6, 22, 4, 1, 6, 8, 9, 86, 3, 4, 32, 64, 10],
+
+    	/*
+	Given an integer array of size N, compute the sum of all even numbers in this array
+	*/
+	firstTest: function () {
+    		var array = this.numbers.slice();
+		//join the elements from array into a string
+		array = array.join();	    	
+		//searches the even numbers from string and returns the matches as an array
+		array = array.match(/\w?(\d+)*[24680]/g);
```

Comment:
What was you thought process when you were working on this solution? Why did you choose to get the even numbers this way instead of just iterating through the array?